packages/core/src/codewhisperer/models/model.ts

```diff
@@ -318,9 +319,31 @@ export class ZipManifest {
     dependenciesRoot: string | undefined = 'dependencies/'
     buildLogs: string = 'build-logs.txt'
     version: string = '1.0'
+    hilCapabilities: string[] = ['HIL_1pDependency_VersionUpgrade']
```

Comment:
Note: Passing along this value in the IDE is how we "feature flag". Seeing this capability will trigger API to send new functionality to API.

Backend changes are already in production.